src/handlers/extract/handler.ts

```diff
@@ -1,12 +1,47 @@
-export const handler = async (): Promise<string | undefined> => {
-  if (
-    String(process.env.EXTRACTION_HANDLER_THROW_ERROR).toLowerCase() === "true"
-  ) {
-    const message =
-      "Environment variable EXTRACTION_HANDLER_THROW_ERROR is true";
-    console.error(message);
-    throw new Error(message);
-  }
-
-  return process.env.EXTRACTION_HANDLER_RETURN_VALUE;
+import * as AWS from "aws-sdk";
+import { S3Event } from "aws-lambda";
+
+export const handler = async (
+  event: S3Event
```

Comment:
This will not be of type S3event. It will be an SQS event.

Reply:
Pending incoming event...